src/core/fonts.js

```diff
@@ -834,6 +834,7 @@ var Font = (function FontClosure() {
       if ((usedFontCharCodes[fontCharCode] !== undefined ||
            isProblematicUnicodeLocation(fontCharCode) ||
            (isSymbolic && !hasUnicodeValue)) &&
+          !missingGlyphs[glyphId] &&
```

Comment:
I think it depends on a number of things and would really need to be benchmarked, but I don't thing this code is particularly hot, so it's not really worth worrying about it. I've updated it to be first though.